deeprank2/utils/buildgraph.py

```diff
@@ -30,57 +30,57 @@ def _add_atom_to_residue(atom: Atom, residue: Residue):
     residue.add_atom(atom)
 
 
-def _add_atom_data_to_structure(structure: PDBStructure,  # pylint: disable=too-many-arguments, too-many-locals
-                                x: float, y: float, z: float,
-                                atom_name: str,
-                                altloc: str, occupancy: float,
-                                element_name: str,
-                                chain_id: str,
-                                residue_number: int,
-                                residue_name: str,
-                                insertion_code: str):
-    """
-    This is a subroutine, to be used in other methods for converting pdb2sql atomic data into a
-    deeprank structure object. It should be called for one atom.
+def _add_atom_data_to_structure(
+    structure: PDBStructure,
+    pdb_obj: pdb2sql_object | pdb2sql_interface,
+    **kwargs
+):
+    """This subroutine retrieves pdb2sql atomic data for `PDBStructure` objects as defined in DeepRank2.
+
+    This function should be called for one atom at a time.
 
     Args:
-        structure (:class:`PDBStructure`): Where this atom should be added to.
-        x (float): x-coordinate of atom.
```

Comment:
are these kwargs (former x, y, x, ... args) described somewhere else? Or maybe they don't add anything here (since `_add_atom_data_to_structure` takes always as input `structure` and `pdb_obj` only)?

Reply:
> are these kwargs (former x, y, x, ... args) described somewhere else? Or maybe they don't add anything here (since `_add_atom_data_to_structure` takes always as input `structure` and `pdb_obj` only)? I wonder why do we need `kwargs` agument at all

the former kwargs, which I now standardized in the `pdb2sql_columns` variable are not really documented properly anywhere. They are used by [this method](https://github.com/DeepRank/pdb2sql/blob/76d7b684b6921261436e2656d9ecd8be86148199/pdb2sql/pdb2sqlcore.py#L403-L587) from pdb2sql and appear are to be the data (columns) to retrieve from the pdb file.

The new `kwargs` that is part of the current `_add_atom_data_to_structure` is the final part of what was previously part of that string. I haven't tested/researched in detail what it does beyond that at least 1 test fails if it's left out. Do you think I should look at it more carefully?